androidApp/src/main/java/com/example/myplants/android/plant/presentation/plantlistscreen/components/PlantItemHolder.kt

```diff
@@ -0,0 +1,175 @@
+package com.example.myplants.android.plant.presentation.plantlistscreen.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.myplants.android.R
+import com.example.myplants.android.core.theme.GrayishBlack
+import com.example.myplants.android.core.theme.Neutrals0
+import com.example.myplants.android.core.theme.Neutrals100
+import com.example.myplants.android.core.theme.Neutrals500
+import com.example.myplants.android.core.theme.Neutrals900
+import com.example.myplants.android.core.theme.OtherG100
+
+@Composable
+fun PlantItemHolder(
+    nextWaterDate: String,
+    plantImageVector: ImageVector?,
+    waterAmount: String,
+    name: String,
+    description: String,
+    isWatered: Boolean
```

Comment:
Wouldn't it make sense to have a class for this information? To me it clearly belongs together, so something like a data class `PlantUi` or so could get rid of all these parameters. Also in case you'd extend this in future, you'd just need to extend the `PlantUi` class and not add a new parameter to all functions that display such a plant